doc/source/visualizing/unstructured_mesh_rendering.rst

```diff
@@ -23,7 +23,7 @@ For conda-based systems, this is trivial, see
 For systems other than conda, you will need to install Embree first, either by
 `compiling from source <https://github.com/embree/embree#installation-of-embree>`_
 or by using one of the pre-built binaries available at Embree's
-`downloads <https://www.embree.org/downloads.html>`_ page.
```

Comment:
interestingly, this one isn't exactly broken, it's just redirected, but I agree we might as well point directly to the final page as you did.